<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1255</width>
    <height>922</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabb">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1251</width>
      <height>881</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_18">
     <attribute name="title">
      <string>Misc.</string>
     </attribute>
     <widget class="QComboBox" name="Map">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>660</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Current map</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>Minish woods</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Minish woods</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Minish village</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hyrule town</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hyrule field</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Castor Wilds</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ruins</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mt. Crenel</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Castle Garden</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cloud Tops</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Royal Valley</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Veil Falls</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lake Hylia</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="Ysmall">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>660</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="Ybig">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>660</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>131</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_81">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>630</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Big y Small y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_82">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>630</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Big x Small x</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Xsmall">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>660</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>248</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="Xbig">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>660</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>131</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_83">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>630</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Area</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="SpwnAnm">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>660</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Spawn animation</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>None</string>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Minish Link</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fall</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>?</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Forward walk</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Minish Link fall</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Stair climb from left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Stair climb from right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Top left scroll</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spin jump forward</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spin jump</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bird fall</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_84">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>630</y>
        <width>251</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Animation</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="customize">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>710</y>
        <width>132</width>
        <height>29</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Customize coordinates as well as spawn animation.</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="text">
       <string>Customize</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_85">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>600</y>
        <width>251</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Coordinates</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="Hatless">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>670</y>
        <width>221</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Hatless (GLITCHES)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_86">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Health</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="Hearts">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_60">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_87">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Hearts</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_88">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Arrows</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Health">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>79</width>
        <height>31</height>
       </rect>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.250000000000000</double>
      </property>
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_89">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Bombs</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="Rupees">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_63">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>70</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_90">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Rupees</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="name">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_91">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>10</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="Brightness">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>110</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_92">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>80</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Brightness</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_93">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>80</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Message Speed</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="Speed">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>110</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_94">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>210</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Heart Pieces</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_64">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>240</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="Wallet">
      <property name="geometry">
       <rect>
        <x>1110</x>
        <y>240</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_95">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>210</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Wallet Upgrade</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="Tri">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>40</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Triforce</string>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="File">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>File 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>File 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>File 3</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="room">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>660</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_96">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>630</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Room</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Flags</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>501</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open a File to start!</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>tabb</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1255</width>
     <height>38</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
